app/controllers/content.js

```diff
@@ -56,7 +56,7 @@ export default Ember.Controller.extend(Analytics, {
     expandedAuthors: true,
     showLicenseText: false,
     fileDownloadURL: '',
-    expandedAbstract: false,
+    expandedAbstract: navigator.userAgent.includes('Prerender'),
```

Comment:
Inital state for prerender is all that matters, because there won't be js that the bot can execute